File: GO/SolutionComplete/Solution1/Web/Web.go
package main

import (
	"fmt"
	"net/http"

	Entities "../Entities"
)

const (
	port = ":8000"
)

var (
	calls = 0
)

//HelloWorld funcion a enrutar
func HelloWorld(w http.ResponseWriter, r *http.Request) {
	var person Entities.Person
	person.FirstName = "David"
	person.LastName = "Silva"

	calls++
	fmt.Fprintf(w, "Hola %s %d veces.\n", person.FirstName, calls)
}

func main() {
	fmt.Printf("Servidor corriendo en http://localhost%v.\n", port)
	http.HandleFunc("/", HelloWorld)
	http.ListenAndServe(port, nil)
}

File: JavaScript/SolutionComplete/Solution1/Web/Web.js
var http = require('http');
var Person = require('../Entities/Person.js');

var person = new Person("David", "Silva");

var port = 8080;
var server = http.createServer(function (request, response) {
    response.writeHead(200, { 'Content-Type': 'text/html' });
    response.write('<h1>Hello ' + person.getFirstName() + '</h1>');
    response.end();
});
server.listen(port, function () {
    console.log('Server working at http://localhost:' + port);
});